util/cron/common.bash

```diff
@@ -101,8 +101,12 @@ export CHPL_NIGHTLY_CRON_LOGDIR=$CHPL_NIGHTLY_LOGDIR
 
 # It is tempting to use hostname --short, but macs only support the short form
 # of the argument.
-export CHPL_TEST_PERF_DIR=$logdir_prefix/NightlyPerformance/$(hostname -s)
-export CHPL_TEST_COMP_PERF_DIR=$logdir_prefix/NightlyPerformance/$(hostname -s)
+if [ -z "$CHPL_TEST_PERF_DIR" ]; then
```

Comment:
Use can use `MY_VAR=${VAR_NAME:-default}` in bash to set $MY_VAR. If $VAR_NAME is defined, it will be used, otherwise the default will be used. For example, this would use the existing CHPL_TEST_PERF_DIR if set, otherwise use the old value as the default:

```bash
export CHPL_TEST_PERF_DIR=${CHPL_TEST_PERF_DIR:-$logdir_prefix/NightlyPerformance/$(hostname -s)}
```

I don't know if I would call it cleaner, but that's how things like $CHPL_NIGHTLY_LOGDIR are set in common.bash.